src/routes/vendor/vendor-show.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import fetch from 'isomorphic-fetch';
+
+import View from '../../components/vendor-show';
+
+class Vendor {
+  constructor(prev = {}, props = {}) {
+    Object.assign(this, prev, props);
+  }
+  get isRequested() { return false; }
+  get isLoading() { return false; }
```

Comment:
i see what state `isLoading` is supposed to represent, but it doesn't appear that we're actually using it anywhere.  we might want to review the logic around these states